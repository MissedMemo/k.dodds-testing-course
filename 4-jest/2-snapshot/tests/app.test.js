import 'react-testing-library/cleanup-after-each'
import React from 'react'
import { render } from 'react-testing-library'
import App from '../src/app'

test( 'renders with expected layout', () => {
  const { container } = render(<App greeting='yowza!' />)
  expect(container.firstChild).toMatchSnapshot() // (firstChild skips snapshotting <div> wrapper)
})