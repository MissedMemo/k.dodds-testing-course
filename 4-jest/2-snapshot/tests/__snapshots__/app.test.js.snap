// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with expected layout 1`] = `
<div>
  <div>
    yowza!
  </div>
</div>
`;
